moped-editor/src/components/GridTable/Filters.js

```diff
@@ -408,8 +415,11 @@ const GridTableFilters = ({
   const handleApplyButtonClick = () => {
     filterQuery.set("filter", btoa(JSON.stringify(filterParameters)));
     history.push(`${queryPath}?filter=${filterQuery.get("filter")}`);
-    filterState.setFilterParameters(filterParameters);
+    setFilters(filterParameters);
     handleAdvancedSearchClose();
+    // Clear simple search field in UI and state since we are using advanced search
+    setSearchFieldValue("");
+    setSearchTerm("");
```

Comment:
This the change that we discussed earlier this week in dev sync. Now, when you click search after setting some advanced filters, the search term is cleared from the UI search box and from the state that is used to build the graphql query.

Reply:
👍 works great